apps/src/blockly/addons/cdoTrashcan.js

```diff
@@ -1,24 +1,236 @@
 import GoogleBlockly from 'blockly/core';
-import {ToolboxType} from '../constants';
 
-export default class CdoTrashcan extends GoogleBlockly.Trashcan {
+export default class CdoTrashcan extends GoogleBlockly.DeleteArea {
+  constructor(workspace) {
+    super();
+    this.workspace = workspace;
+    this.id = 'cdoTrashcan';
+
+    /**
+     * Width of both the trash can and lid images.
```

Comment:
I admire the comment formatting! Since this ends up taking a bunch of space at the beginning of the file, I wonder if it would be appropriate to move to a constants file (like: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/blockly/constants.js).

Reply:
Discussed over video and then [Slack](https://codedotorg.slack.com/archives/C0T0PNR0D/p1652206954558049). Updated to follow the conventions established by the Google Blockly team.